scss/custom.scss

```diff
@@ -76,39 +76,37 @@ h6 {
 }
 
 // Import all partials & page css files
-@import 'index.scss';
-@import 'navbar.scss';
-
+@import "index.scss";
+@import "navbar.scss";
 
 .slider {
-     width: 105%;
-     height: 100%;
-     margin: 100px auto;
- }
-
- .slick-slide {
-   margin: 0px 20px;
- }
+    width: 105%;
+    height: 100%;
+    margin: 100px auto;
+}
 
- .slick-slide img {
-   width: 100%;
- }
+.slick-slide {
+    margin: 0px 20px;
+}
 
- .slick-prev:before,
- .slick-next:before {
-   color: black;
- }
+.slick-slide img {
+    width: 100%;
+}
 
+.slick-prev:before,
+.slick-next:before {
+    color: black;
```

Comment:
We might need to create a .eslintrc file to lay down format standards, will help reduce diffs like these @geooot